docs/users/cli-usage.md

```diff
@@ -57,13 +57,13 @@ ddev config
 
 _Note: ddev config will prompt you for a project name, docroot, and project type._
 
-After you've run `ddev config` you're ready to start running your project. Run ddev using:
+After you've run `ddev config`, you're ready to start running your project. Run ddev using:
```

Comment:
I think all these "After you've run" statements are awkward. Maybe just "After `ddev config`"?

Reply:
Yes, and it was inconsistent too. Fixed!